README.md

```diff
@@ -1,26 +1,26 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/newrelic/newrelic-istio-adapter)](https://goreportcard.com/report/github.com/newrelic/newrelic-istio-adapter)
 [![Build Status](https://travis-ci.com/newrelic/newrelic-istio-adapter.svg?branch=master)](https://travis-ci.com/newrelic/newrelic-istio-adapter)
 
-# New Relic Istio Adapter
+# New Relic Istio adapter
```

Comment:
I prefer using title case for section headers. Is this case structure a standard we are trying to adhere to?